nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters
# Delete working on Comments

# Description
As a commenter I would like the ability to remove a Comment so that I have created so that I can prevent others from seeing it in the event that I regret what I said.

Given a user is vising the list of Comments
When they select the Delete option
Then they should be prompted to confirm the deletion

Given the user wishes to confirm the delete
When they the select the option to confirm
Then the Comment should be removed from the system
And the user should be directed back to the Comment list

Given the user decides not to confirm the delete
When they the select the option to reject confirmation
Then the Comment should NOT be removed from the system
And the user should be directed back to the Comment list
# Ticket Reference

> [#21- Delete A Comment](https://github.com/nss-day-cohort-41//tabloid-fullstack-crop-circle-cavorters/21)


## Type of change


# Testing Instructions

- Start program
- Go to localhost:3000 in your browser.
- Login with "duck@wiz.com"
- Navigate to the "Posts" link in the navbar.
- Locate The comments button and click the "Comments" button in the center.
- First click the back to posts button and ensure the link brings you back to the post page, then repeat the previous step.
- Click Delete on the top comment to delete individual comment and be directed to a "Are you sure you want to delete page"
- Visually confirm that you are on the "Delete" form page. 
- Next, click the delete button on the bottom of the form.
-Visually confirm the comment has been deleted.
-Return to Posts page
-Logout user.

# Checklist:
